File: Conference-Room-Reservation-System/Conference-Room-Reservation-System/src/main/java/com/sda/conferenceroomreservationsystem/model/entity/Reservation.java
package com.sda.conferenceroomreservationsystem.model.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Reservation {
    @Id
    @GeneratedValue
    private Long reservationId;
    @Column(unique = true)
    private String reservationIdentifier;
    private LocalDateTime reservationStart;
    private LocalDateTime reservationEnd;
    @ManyToOne
    @JoinColumn(name = "conferenceRoomId")
    private ConferenceRoom conferenceRoom;
}

File: Front-Conference-Room-Reservation-System/Front-Conference-Room-Reservation-System/src/app/model/Reservation.ts
export class Reservation{

    reservationIdentifier:string;
    reservationStart: Date;
    reservationEnd: Date;

    constructor(reservationIdentifier:string,reservationStart: Date,reservationEnd: Date){
        this.reservationIdentifier = reservationIdentifier;
        this.reservationStart = reservationStart;
        this.reservationEnd = reservationEnd;

    }
}